setup.cfg

```diff
@@ -35,8 +35,8 @@ python_requires = >=3.7
 install_requires =
     graphviz>=0.12.0
     particle>=0.16
-    awkward>=1.2.0
-    vector>=0.8.1
```

Comment:
`vector` `v0.11.0` only requires `awkward>=1.2` for the `awkward` extra. 

https://github.com/scikit-hep/vector/blob/eb2b4142a28aacc3666197ba2c822b235e4c391b/pyproject.toml#L49-L51

Why do we need to bump the lower bound here? Is there something that breaks with `vector==0.8.1`?

Reply:
There are fair points. To be honest I took the update from the scikit-hep metapackage where I ping =~ versions, and make consistent versions moving forward as the dependent packages get newer releases.

I realise the lower bounds ensure awkard v2, etc., are used. I thought I would make it explicit that we are moving to v2, hence this change. It's a different angle of thought from you.

This also explains the change in vector - not compulsory given the lower bound but it makes it clear that we take a version that supports awkward 2 (while keeping support for v1), see https://github.com/scikit-hep/vector/releases/tag/v0.11.0.

Up to you. I have no strong feelings. If you prefer we go with the trivial scikit-hep-testdata update and nothing else. Let me know.

Reply:
OK, I will reverse that part of this MR, then.